sass/_Quickview.scss

```diff
@@ -20,23 +20,42 @@
     z-index: 1;
     > div {
       border: solid;
-      border-color: $color-greyish-dark-blue transparent;
       content: '';
       position: absolute;
       z-index: 99;
-      left: calc(50% - 6px);
     }
 
     // Tooltip under element, arrow on top
     &[x-placement^='bottom'] > div {
       top: -6px;
+      left: calc(50% - 6px);
       border-width: 0 6px 7px;
+      border-color: $color-greyish-dark-blue transparent;
     }
 
     // Tooltip over element, arrow on bottom
     &[x-placement^='top'] > div {
       bottom: -6px;
+      left: calc(50% - 6px);
       border-width: 7px 6px 0;
+      border-color: $color-greyish-dark-blue transparent;
+    }
+
+    // Tooltip on the right of element, arrow on the left
+    &[x-placement^='right'] > div {
+      top: calc(50% - 5px);
```

Comment:
Indeed why do we have a -6px on each position ?

Reply:
It will handle the arrow positioning as long as we place it in the middle correctly initially (it will offset it from there)
Positions and border-width is for the arrow (a 6px width, maybe I should put it in a variable)

Take a look at was popper themselves do in their docs
https://github.com/FezVrasta/popper.js/blob/master/docs/css/popper.css#L81